README.md

```diff
@@ -80,6 +80,50 @@ case object GetPeople extends CollectionQuery[Seq, Person] {
 val person = db(GetPeople).head // Person(Coda Hale,chale@example.com,29)
 ```
 
+You can also use the sql string interpolator:
```

Comment:
+1 on @jklukas suggestion of re-working the docs to highlight the pro's/con's of using interpolation.  

@vanvlack would you mind filing an issue internally on open sourcing our metrics lib?